<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagramText</class>
 <widget class="QMainWindow" name="DiagramText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiagramText</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>8</y>
      <width>524</width>
      <height>456</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>parse info text</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="imageLoc">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="imageParse">
           <property name="text">
            <string>Parse Image</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QListWidget" name="imgeleListWidget"/>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QTextEdit" name="QtextLoc"/>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="textParseBtn">
           <property name="text">
            <string>Parse Text</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="textFilterBtn">
           <property name="text">
            <string>Filter Text</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="wordObjectList"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>speech</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionTextImport"/>
    <addaction name="actionImageImport"/>
    <addaction name="actionImport_Image_Informations"/>
   </widget>
   <addaction name="menuImport"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTextImport">
   <property name="text">
    <string>Import Question Text</string>
   </property>
  </action>
  <action name="actionImageImport">
   <property name="text">
    <string>Import Question Image </string>
   </property>
  </action>
  <action name="actionImport_Image_Informations">
   <property name="text">
    <string>Import Image Informations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
